<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hobby Roll</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>561</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="roll">
    <attribute name="title">
     <string>Рулетка</string>
    </attribute>
    <widget class="QLabel" name="roll_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>531</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Крути</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="roll_btn">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Крутить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings">
    <attribute name="title">
     <string>Настройки</string>
    </attribute>
    <widget class="QPushButton" name="add_btn">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_hobby">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>351</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="hobby_list">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>441</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="del_btn">
     <property name="geometry">
      <rect>
       <x>404</x>
       <y>310</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
